vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -185,6 +187,32 @@ - (BOOL)shouldAllowAdConversionTracking
   return true;
 }
 
+- (const std::string)getCountrySubdivision
+{
+  // Not needed on mobile
```

Comment:
Is this feature only for desktop?

Reply:
that was a placeholder forgot to revisit the iOS bits, will push with the next commit